### DISCUSSION TITLE
v4.1 release

### DISCUSSION THREAD

**Discussion by WouterVdS** [axelboc/anki-ultimate-geography#380] [General]
Hi,

What is the plan on bringing out new releases? Seems like with the addition of Czech, Russian and Dutch it seems like a good time!

**Comment by axelboc** [comment]
Totally, I'll do a pre-release soon after #379 is merged 👍

**Reply by aplaice** [reply]
I assume that we won't be removing the White Sea and the Denmark Strait just before the new release? It'd be a bit of a shame and a slight waste of translation effort! :)

**Reply by aplaice** [reply]
Also, just in in case, it seems that both of GitHub's command-line applications allow uploading release files via the CLI:

[hub release](https://manpages.debian.org/buster/hub/hub-release.1.en.html)

[gh release create](https://cli.github.com/manual/gh_release_create)

(I think you had been unhappy about having so many files to upload, at the previous release, and now it'll be ~ 17 files...)

**Reply by axelboc** [reply]
Ooh I'll keep that in mind, thanks! I may actually upload one zip per language with two decks inside each, I don't know. I definitely need to automate my release process a little bit more.

**Comment by WouterVdS** [comment]
Awesome!

**Comment by axelboc** [comment]
The pre-release for v4.1 is out!! https://github.com/axelboc/anki-ultimate-geography/releases/tag/v4.1-beta

Please upgrade and report any issues. If everything looks alright in a week, I'll publish the official release.

**Reply by aplaice** [reply]
I've imported the deck over my current collection and tested importing the English `.apkg` and the Russian `CrowdAnki` deck (Russian selected randomly). I did not encounter any issues.

**Reply by jeprecated** [reply]
хорошая работа, товарищ!
